salt/cloud/clouds/ec2.py

```diff
@@ -216,7 +216,7 @@ def _xml_to_dict(xmltree):
             name = comps[1]
         if name not in xmldict:
             if sys.version_info < (2, 7):
-                children_len = len(item.getchildren())
+                children_len = len(list(item))
```

Comment:
This whole `if` block should be removed. Salt doesn't support Py2 anymore, let alone 2.6.

Just use the `else` clause, `children_len = len(item)`